blueprints/codes/eurocode/nen_en_1993_1_1_c2_a1_2016/chapter_6_ultimate_limit_state/formula_6_5.py

```diff
@@ -0,0 +1,44 @@
+"""Formula 6.5 from NEN-EN 1993-1-1+C2+A1:2016: Chapter 6 - Ultimate limit state."""
+# pylint: disable=arguments-differ
+from blueprints.codes.eurocode.nen_en_1993_1_1_c2_a1_2016 import NEN_EN_1993_1_1_C2_A1_2016
+from blueprints.codes.formula import Formula
+from blueprints.type_alias import KN
+
+
+class Form6Dot5UnityCheckTensileStrength(Formula):
+    """Class representing formula 6.5 for the unity check for tensile strength."""
+
+    label = "6.5"
+    source_document = NEN_EN_1993_1_1_C2_A1_2016
+
+    def __init__(
+        self,
+        n_ed: KN,
+        n_t_rd: KN,
+    ) -> None:
+        """[N_ed/N_t_rd] Unity check for tensile strength of an element in tension.
+
+        NEN-EN 1993-1-1+C2+A1:2016 art.6.2.3(1) - Formula (6.5)
+
+        Parameters
+        ----------
+        n_ed : KN
+            [NEd] Design value of the normal tensile force [kN].
+        n_t_rd : KN
+            [Nt,Rd] Design value of the resistance against tenslie force [kN].
+        """
+        super().__init__()
+        self.n_ed = n_ed
+        self.n_t_rd = n_t_rd
+
+    @staticmethod
+    def _evaluate(
+        n_ed: KN,
+        n_t_rd: KN,
+    ) -> float:
```

Comment:
```suggestion
    ) -> RATIO:
```
see bluprints/type_alias.py

Reply:
Fixed in https://github.com/Blueprints-org/blueprints/commit/9dc52fbbfe22a3df3a0d355570a8611245980a7b 
The rest is still a work in progress.